lib/Notes/NotesAccordion/NotesAccordion.js

```diff
@@ -0,0 +1,129 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+} from 'react-intl';
+
+import {
+  Accordion,
+  Headline,
+  Badge,
+  Button,
+} from '@folio/stripes-components';
+
+import NotesList from './components';
+import styles from './NotesAccordion.css';
+
+const defaultColumnsConfig = [
+  {
+    name: 'date',
+    title: <FormattedMessage id="stripes-smart-components.date" />,
+    width: '30%',
+  },
+  {
+    name: 'updatedBy',
+    title: <FormattedMessage id="stripes-smart-components.updatedBy" />,
+    width: '30%',
+  },
+  {
+    name: 'title',
+    title: <FormattedMessage id="stripes-smart-components.title" />,
+    width: '40%',
+  },
+];
+
+export const columnShape = PropTypes.shape({
+  name: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired,
```

Comment:
This should be `PropTypes.node`, instead of `PropTypes.string` right? due to the `<FormattedMessage>` use here.